code/cutscene/cutscenes.cpp

```diff
@@ -35,6 +35,7 @@ const char* Cutscene_mask_name[GR_NUM_RESOLUTIONS] = {
 		"2_ViewFootage-m"
 };
 
+int Num_cutscenes;
```

Comment:
Since `Cutscenes` is a vector, a `Num_cutscenes` variable is redundant.  Change the loop comparison to `i < (int)Cutscenes.size()`, and change the pointer assignment to `csnp = &Cutscenes[Cutscenes.size() - 1];` or `csnp = &Cutscenes.back();`